src/components/Reports/FinancialAccountsReport/AccountTransactions/AccountTransactions.tsx

```diff
@@ -0,0 +1,196 @@
+} from '../Context/FinancialAccountsContext';
+import { FinancialAccountHeader } from '../Header/Header';
+import {
+  AccountTransactionTable,
+  FinancialAccountEntryTypeEnum,
+} from './AccountTransactionTable/AccountTransactionTable';
+import { useFinancialAccountEntriesQuery } from './financialAccountTransactions.generated';
+
+const Container = styled(Box)(() => ({
+  height: `calc(100vh - ${headerHeight})`,
+  overflowY: 'auto',
+}));
+
+const formatDateRange = (startDate?: DateTime, endDate?: DateTime) => {
+  if (!startDate) {
+    startDate = DateTime.local().minus({ months: 1 }).plus({ days: 1 });
+  }
+  if (!endDate) {
+    endDate = DateTime.local();
+  }
+  return `${startDate.toISODate()}..${endDate.toISODate()}`;
+};
+
+const defaultDateRange = formatDateRange();
+const defaultStartDate = defaultDateRange.split('..')[0];
+const defaultEndDate = defaultDateRange.split('..')[1];
+
+export const AccountTransactions: React.FC = () => {
+  const { query } = useRouter();
+  const { t } = useTranslation();
+  const locale = useLocale();
+  const {
+    accountListId,
+    financialAccountId,
+    activeFilters,
+    setActiveFilters,
+    hasActiveFilters,
+    searchTerm,
+    setPanelOpen,
+  } = useContext(FinancialAccountContext) as FinancialAccountType;
+  const { appName } = useGetAppSettings();
+
+  useEffect(() => {
+    // On loading the transactions page, open the filters panel and reset the active filters.
+    // We need to reset the active filters to ensure the date range is set to the date range in the URL if it exists.
+    const urlFilters =
+      query?.filters && JSON.parse(decodeURI(query.filters as string));
+    setPanelOpen(Panel.Filters);
+    setActiveFilters(urlFilters ?? {});
+    return () => {
+      setPanelOpen(null);
+      setActiveFilters({});
+    };
+  }, [query?.filters]);
+
+  useEffect(() => {
+    if (!hasActiveFilters && !query?.filters) {
+      setActiveFilters({
+        dateRange: {
+          min: defaultStartDate,
+          max: defaultEndDate,
+        },
+      });
+    }
+  }, [hasActiveFilters]);
+
+  const dateRange = useMemo(() => {
+    if (!activeFilters?.dateRange?.min || !activeFilters?.dateRange?.max) {
+      return defaultDateRange;
+    }
+    return formatDateRange(
+      DateTime.fromISO(activeFilters.dateRange.min),
+      DateTime.fromISO(activeFilters.dateRange.max),
+    );
+  }, [activeFilters]);
+  const categoryId =
+    activeFilters?.categoryId && activeFilters?.categoryId !== 'all-categories'
+      ? activeFilters?.categoryId
+      : '';
+  const wildcardSearch = useDebouncedValue(searchTerm, 500);
+
+  const { data } = useFinancialAccountEntriesQuery({
+    variables: {
+      input: {
+        accountListId,
+        financialAccountId: financialAccountId ?? '',
+        dateRange,
+        categoryId,
+        wildcardSearch,
+      },
+    },
+  });
+
+  const handleExportCSV = () => {
+    if (!data) {
+      // Alert user there is no data
```

Comment:
Maybe we should pass a prop to `Header` to notify it that data is loading so that it can disable the export button?

Reply:
Ah sorry I missed this one